GameServerLib/Items/Shop.cs

```diff
@@ -56,55 +54,19 @@ public bool HandleItemBuyRequest(int itemId)
                     return false;
                 }
 
-                foreach (var item in ownedItems)
+                foreach (var itens in ownedItems)
```

Comment:
Should be `items`

Reply:
> Generally approved. I'll have to test this and check if it works. I'll open an issue about passing the owner as parameter. I think that a class variable is the correct solution (and also merging InventoryManager with Inventory at some point as it is not clear why these two got separated). Is there any scenario that owner is actually needed to be `null`?

I don't remember any actual case in-game they were `null`. I added it because`TestInventoryManager.cs` doesn't contain have any IObjAiBase to be use as an owner, so i thought it would be best to got the "safe" way and make an exeception for it